src/vm/util.cpp

```diff
@@ -1861,8 +1861,6 @@ size_t GetLargestOnDieCacheSize(BOOL bTrueSize)
     STATIC_CONTRACT_NOTHROW;
     STATIC_CONTRACT_GC_NOTRIGGER;
 
-#if defined(_TARGET_AMD64_) || defined (_TARGET_X86_)
-
     static size_t maxSize;
     static size_t maxTrueSize;
```

Comment:
There is no need for volatile, as static local variables are guaranteed to be initialized in a thread safe manner. See http://en.cppreference.com/w/cpp/language/storage_duration#Static_local_variables